gqueries/output_elements/output_series/vertical_stacked_bar_207_synthetic_kerosene_production/transport_final_demand_kerosene.gql

```diff
@@ -0,0 +1,9 @@
+- unit = PJ
+- query =
+    DIVIDE(
+        SUM(
+            V(bunkers_useful_demand_planes,demand),
```

Comment:
Just me not understanding energy maybe, but isn't there a `final_demand` for bunkers?

Reply:
Yes you are right! But this `final_demand` is varying for kerosene when synthetic kerosene is produced and that should not be the case in this chart. Therefore I chose to use the useful demand in this chart.